src/ducks/categories/CategoriesPage.jsx

```diff
@@ -216,7 +223,7 @@ export class CategoriesPage extends Component {
             ) : (
               <CategoryTransactions
                 subcategoryName={router.params.subcategoryName}
-                transactions={transactions.data}
+                transactions={filteredTransactionsByAccount}
```

Comment:
It makes me feel weird to add a new props to this component and only use it in a certain case. 

Also we are in `isFetching` state when this transactionsByApplicationDate has not been computed even if we don't use it for instance when we display `Categories` component. 

Shouldn't we make the call inside the `CategoryTransactions` instead of this top level component? For someone external to the project this component since to have a lot of props and seems pretty complex to understand

Reply:
Les `transactionsData` (qui sont finalement passé à `CategoryTransactions`) sont aussi nécessaire pour savoir quelles `categories` on doit afficher, `categories` qui sont passé aux comp `CategoriesHeader` et `Categories`. Du coup on fait bien tout le traitement dans le parent pour passer ensuite les résultats en prop, ça me paraît pertinent non ? Sinon il faudrait dupliquer toute la logique dans `CategoryTransactions`.

Par contre il y a de la logique dans `CategoriesPage` et `enhanceCategoriesPage`, peut-être qu'on peut tout mettre dans `enhanceCategoriesPage` et passer des props "prêtes à être consommées" à `CategoriesPage` ?